### ISSUE TITLE
VideoPress: Warning about max media size when adding media

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#5986]
### Steps to reproduce

0. Start on a site where VideoPress is active.
1. Go to Posts > Add New
2. Click on Add Media
3. Click on the Upload tab.

![screen shot 2016-12-22 at 23 08 28](https://cloud.githubusercontent.com/assets/426388/21442031/a8a68cc8-c89b-11e6-897f-435a04c95cb9.png)

Since I'm using VideoPress, that's not true for videos. It's confusing for folks since they may think they'll need to use a different interface for videos.

**Comment by dbtlr** [comment]
Agreed that it is confusing. Not sure if this is changeable or if it is,
how we should change the messaging to have this make more sense.

**Comment by aheckler** [comment]
3069977-t was an instance of this, though the user has since stopped using VideoPress

**Comment by jeherve** [comment]
Also in 937773-zen

**Comment by jeherve** [comment]
Also in 937460-zen
